qhub/provider/cicd/github.py

```diff
@@ -215,7 +215,7 @@ def gen_qhub_ops(config):
 
     env_vars = gha_env_vars(config)
     branch = config["ci_cd"]["branch"]
-    commit_render = config["ci_cd"]["commit_render"]
+    commit_render = config["ci_cd"].get("commit_render", "")
```

Comment:
```suggestion
    commit_render = config["ci_cd"].get("commit_render", True)
```

Reply:
My bad, I forgot that the default value should be True

Reply:
I realized that it has a default set to True elsewhere but this will just make sure of it ;)